src/app/content/components/BuyBook.tsx

```diff
@@ -47,13 +47,11 @@ const BuyPrintDisclosure = styled.p`
 // tslint:disable-next-line: variable-name
 const BuyBook = () => {
   const config = useSelector(buyPrintConfig);
-  const formatMessage = useIntl().formatMessage;
 
   if (!config) { return null; }
 
   return <BuyBookAlignment>
     <BuyBookLink
-      aria-label={formatMessage({id: 'i18n:toolbar:buy-book:aria-label:text'})}
```

Comment:
It doesn't need aria-label, since it has a text value.
Lighthouse didn't like the text value being different from the aria-label value.

Reply:
I also like that the normal text is more descriptive to the screen reader as well. This appears to be the only thing that used it, can we delete the key from the messages files too?